### PR TITLE
Animals table

### PR DESCRIPTION
**Pull request by joyapisi**
# What I did:
- Create  animals table
-  Insert data to animals table
- Query animals table:
  - Find all animals whose names ends in "mon

  - List the name of all animals born between 2016 and 2019. 

  - List the name of all animals that are neutered and have less than 3 escape attempts

  - List the date of birth of all animals named either "Agumon" or "Pikachu".

  - List name and escape attempts of animals that weigh more than 10.5kg

  - Find all animals that are neutered

  - Find all animals not named Gabumon

  - Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -286,6 +183,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (t-yanick) [inline README.md]: - [ ] Great job working on this milestone. When I click on the MIT license link, I am re-directed to an error 404 page. Kindly fix this issue as it is not professional to allow broken links in our README.
image

File: schema.sql [hunk 2]
```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id int PRIMARY KEY NOT NULL,
+    name text NOT NULL,
```
Comment (t-yanick) [inline schema.sql]: - [ ] Kindly use `VARCHAR` for the name column and specify a length. example` name varchar(255)`.
`VARCHAR` is best suited for storing short to medium-length `strings`, while `TEXT` is better suited for storing large amounts of textual data. As you can clearly see from the project requirements, the data type recommended for name column is string.
image

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @joyapisi , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark: SQL commands put in the correct files
:heavy_check_mark:  Good commit messages
:heavy_check_mark: All queries added and given required results

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- [x] The requirements said you should take screenshots and put in the PR description and not providing links to the images. Just as I have added a screenshot below is exactly what you were expected to do. I have not just provided a link to the image.
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @joyapisi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**